StrongerUITests/HomeTest.swift

```diff
@@ -71,9 +74,12 @@ class HomeTests: XCTestCase {
         app.collectionViews.staticTexts["License Information"].tap()
         app.navigationBars.buttons.element(boundBy: 0).tap()
 
-        app.navigationBars.buttons["Close"].tap()
+        let canClose = app.navigationBars.buttons["Close"].exists
```

Comment:
This should be unnecessary. I think the tests here were failing because of some mistake by the testing harness. Either way is a good addition.